recipe/meta.yaml

```diff
@@ -12,51 +15,82 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 2
+  skip: true            # [win32]
+  skip: true            # [win and py<35]
+  skip: true            # [cuda_compiler_version not in (undefined, "None", "9.2")]
+  number: {{ number }}
 
-requirements:
-  build:
-    - cmake
-    - ninja
-    - make  # [unix]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-  host:
-    # directly pin boost-cpp as we also seem to directly include boost symbols
-    # in the Python modules.
-    - arrow-cpp {{ version }}
-    - boost-cpp
-    - cython
-    - numpy 1.16.*
-    - python
-    - setuptools
-    - setuptools_scm
-    - six
+outputs:
```

Comment:
Why do you need a outputs here? If that is removed this diff will be much smaller and easier to review

Reply:
You can ignore whitespace-changes in the diff using ⚙️ -> "Hide whitespace changes" above...

Reply:
I agree that using `outputs` is not needed for this PR. I'll eliminate it..

Reply:
Thanks for the invite @pearu. If you want, I'll wait for you to remove the outputs section, or do you want me to do that?

Reply:
I would likely have time for this tomorrow but it's fine with me for you to eliminate outputs :)

Reply:
It would make sense to do the clean up after the main issue with cuda-compat is resolved.

Reply:
> It would make sense to do the clean up after the main issue with cuda-compat is resolved.

I had started on this before I saw you second message, sorry. I pushed to the state of how I'd hope the recipe can be, but it is still pending the two PRs I mentioned.